pkg/server/ca/manager_test.go

```diff
@@ -208,25 +210,27 @@ func (s *ManagerSuite) TestUpstreamIntermediateSignedWithUpstreamBundle() {
 	})
 	s.initUpstreamSignedManager(upstreamAuthority, true)
 
+	root := upstreamAuthority.UpstreamAuthority.(*fakeupstreamauthority.UpstreamAuthority).Root()
```

Comment:
These changes can all be reverted if you follow the suggestion in fakeupstreamauthority.New.